config/config.go

```diff
@@ -345,38 +341,19 @@ func (cfg *Routes) validateAndLog() {
 	}
 }
 
-type Stats struct {
-	StatsHost   string `mapstructure:"host"`
-	StatsPort   string `mapstructure:"port"`
-	StatsDCName string `mapstructure:"dc_name"`
-
-	PortTCP                   string `mapstructure:"port_tcp"`
-	PublishInterval           int    `mapstructure:"publish_interval"`
-	PublishThreshold          int    `mapstructure:"publish_threshold"`
-	Retries                   int    `mapstructure:"retries"`
-	DialTimeout               int    `mapstructure:"dial_timeout"`
-	KeepAliveDuration         int    `mapstructure:"keep_alive_duration"`
-	MaxIdleConnections        int    `mapstructure:"max_idle_connections"`
-	MaxIdleConnectionsPerHost int    `mapstructure:"max_idle_connections_per_host"`
-
-	UseTCP bool `mapstructure:"use_tcp"`
-}
-
-func (cfg *Stats) validateAndLog() {
-	logger.Info("config.stats.host: %s", cfg.StatsHost)
-	logger.Info("config.stats.port: %s", cfg.StatsPort)
-	logger.Info("config.stats.dc_name: %s", cfg.StatsDCName)
-
-	logger.Info("config.stats.port_tcp: %s", cfg.PortTCP)
-	logger.Info("config.stats.publisher_interval: %d", cfg.PublishInterval)
-	logger.Info("config.stats.publisher_threshold: %d", cfg.PublishThreshold)
-	logger.Info("config.stats.retries: %d", cfg.Retries)
-	logger.Info("config.stats.dial_timeout: %d", cfg.DialTimeout)
-	logger.Info("config.stats.keep_alive_duration: %d", cfg.KeepAliveDuration)
-	logger.Info("config.stats.max_idle_connections: %d", cfg.MaxIdleConnections)
-	logger.Info("config.stats.max_idle_connections_per_host: %d", cfg.MaxIdleConnectionsPerHost)
-
-	logger.Info("config.stats.use_tcp: %t", cfg.UseTCP)
+func validateAndLogStats(stats stats.StatsConfig) {
```

Comment:
can we use "%+v" formative ? it can print all variable names and respective values ?
https://go.dev/play/p/7WyGjOkWrRl

Reply:
we need to print keys like `config.stat.*` with its value to make it consistent with older approach.